test/geojsonm/expect/testIters.ml

```diff
@@ -0,0 +1,39 @@
+let capitalise_nom obj =
```

Comment:
Can you name the file `test_iters` and also add a [test stanza](https://dune.readthedocs.io/en/stable/dune-files.html#test) for this code (at the moment this code is not being compiled or run I think).

Reply:
Please also delete this file :))